js/modules/migrations/run_migrations.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-env browser */
+
+const isFunction = require('lodash/isFunction');
+const isObject = require('lodash/isObject');
+const isString = require('lodash/isString');
+const head = require('lodash/head');
+const last = require('lodash/last');
```

Comment:
I know of two reasons to require `lodash` this way:

1) It makes webpack bundles more efficient (if you're not using https://www.npmjs.com/package/babel-plugin-lodash)
2) It makes it very clear which functions you're using downfile

Is there any other reason to do this? Would you be open to something like this?

```
const {
  isFunction,
  isObject,
  isString,
  head,
  last,
} = require('lodash');
```

Seems like it would be a little less repetitive. And we don't have that webpack requirement.